src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -56,7 +58,7 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
         parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editPersonDescriptor::setTags);
-
+        parseChildTagsForEdit(argMultimap.getAllValues(PREFIX_CHILD)).ifPresent(editPersonDescriptor::addAllTags);
```

Comment:
Perhaps this line can be broken down into smaller parts for better code readability?